File: server/routes/secretRoutes.js
const express = require('express');
const secretController = require('../controllers/secretController')
const { userVerification } = require('../middlewares/authMiddleware');
const router = express.Router()

router.get('/', secretController.getSecrets)

router.post('/', userVerification, secretController.postSecret)

router.put('/', userVerification, secretController.updateSecret)

module.exports = router;

File: go-server/routes/secretRoutes.go
package routes

import (
	"letItOut/controllers"

	"github.com/gorilla/mux"
)

func setupSecretRoutes(r *mux.Router) {
	r.HandleFunc("", controllers.GetSecrets).Methods("GET")
	r.HandleFunc("", controllers.PostSecret).Methods("POST")
	r.HandleFunc("", controllers.UpdateSecret).Methods("PUT")
}
